src/app/child-dev-project/children/children-list/bmi-block/bmi-block.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { HealthCheck } from "app/child-dev-project/health-checkup/model/health-check";
+import { Database } from "app/core/database/database";
+import { MockDatabase } from "app/core/database/mock-database";
+import { DatabaseIndexingService } from "app/core/entity/database-indexing/database-indexing.service";
+import { EntityMapperService } from "app/core/entity/entity-mapper.service";
+import { EntitySchemaService } from "app/core/entity/schema/entity-schema.service";
+import { of } from "rxjs";
+import { ChildrenService } from "../../children.service";
+import { Child } from "../../model/child";
+
+import { BmiBlockComponent } from "./bmi-block.component";
+
+describe("BmiBlockComponent", () => {
+  let component: BmiBlockComponent;
+  let fixture: ComponentFixture<BmiBlockComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [BmiBlockComponent],
+      providers: [
+        { provide: Database, useClass: MockDatabase },
+        EntityMapperService,
+        EntitySchemaService,
+        DatabaseIndexingService,
+        ChildrenService,
```

Comment:
it would be sufficient to only mock the ChildrenService and omit the other dependencies